Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/PublicPageController.java

```diff
@@ -0,0 +1,100 @@
+package edu.asu.spring.quadriga.web.workbench;
+
+import java.security.Principal;
+import java.util.Properties;
+
+import javax.annotation.Resource;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.InitBinder;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.spring.quadriga.domain.IUser;
+import edu.asu.spring.quadriga.domain.factory.workbench.IProjectFactory;
+import edu.asu.spring.quadriga.domain.impl.workbench.Project;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.IUserManager;
+import edu.asu.spring.quadriga.service.workbench.IModifyProjectManager;
+import edu.asu.spring.quadriga.validator.PublicPageValidator;
+
+@Controller
+public class PublicPageController {
```

Comment:
I'm confused, why does the PublicPageController has method to create a new project?